core/src/main/scala/scalafix/util/ChangeMethod.scala

```diff
@@ -0,0 +1,41 @@
+package scalafix.util
+
+import scala.collection.immutable.Seq
+import scala.{meta => m}
+import scalafix.rewrite.{Desugared, RewriteCtx, SemanticApi}
+
+case class ReplaceTerm(original: m.Type, newString: String)
+
+class ChangeMethod(ast: m.Tree, ctx: RewriteCtx)(implicit sApi: SemanticApi) {
+  object DType extends Desugared[m.Type]
+  object DTerm extends Desugared[m.Term]
+
+  //This isn't very satisfying.  I am trying to match on a method, without including
+  //it's arguments
+  def partialTermMatch(rt: ReplaceTerm)
+    : PartialFunction[m.Tree, (scala.meta.Term, ReplaceTerm)] = {
+    case t @ DTerm(desugared)
+        if desugared.syntax.startsWith(rt.original.syntax) &&
+          desugared.syntax.endsWith("]") =>
```

Comment:
I had to resort to string matching here. If anyone has better ideas I would love the feedback.